libs/sdk-core/src/persist/migrations.rs

```diff
@@ -427,6 +427,11 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
        ALTER TABLE swaps_info ADD COLUMN total_incoming_txs INTEGER;
        UPDATE swaps_info SET status = 0;
        ",
+       // Add max absolute value payable by the swapper. For existing swaps, initialize it to max_allowed_deposit.
+       "
+       ALTER TABLE sync.swaps ADD COLUMN max_swapper_payable INTEGER NOT NULL;
```

Comment:
Will this `NOT NULL` cause a constraint error if some swaps rows already exist?

Reply:
Good point. @ok300 perhaps we should add default zero here?